demisto_sdk/commands/content_graph/objects/script.py

```diff
@@ -38,3 +49,53 @@ def imported_by(self) -> List[BaseContent]:
             for r in self.relationships_data[RelationshipType.IMPORTS]
             if r.content_item_to.database_id == r.source_id
         ]
+
+    def dump(self, dir: DirectoryPath, marketplace: MarketplaceVersions) -> None:
+        dir.mkdir(exist_ok=True, parents=True)
+        data = self.prepare_for_upload(current_marketplace=marketplace)
+
+        for data in MarketplaceIncidentToAlertScriptsPreparer.prepare(
+            data, marketplace, self.is_incident_to_alert(marketplace)
+        ):
+            # Two scripts return from the preparation, one the original, and other the new script,
+            # in order to normalize the name of the new script, make a copy of the original object
+            # in case it is a new script with an update of the name and path.
+            script_name = data.get("name")
+            
+            # In case the original script
+            if script_name == self.name:
+                obj = self
+            # In case the new script
+            else:
+                obj = self.copy(
```

Comment:
```suggestion

            if script_name == self.name:  # the original script
                obj = self

            else:  # a modified script, replaced incidents->alerts
                obj = self.copy(
```
Moving the comment just after the `if` statement makes it, imo, more noticable